packages/filebrowser/src/browser.ts

```diff
@@ -394,6 +399,22 @@ export class FileBrowser extends SidePanel {
     }
   }
 
+  /**
+   * Given a drive name and a local path
+   *
+   * @param driveName: the name of the drive
+   * @param localPath: the local path on the drive.
+   *
+   * @returns the fully qualified path.
+   */
+  private _toGlobalPath(driveName: string, localPath: string): string {
```

Comment:
The docstring says `@returns the fully qualified path.` and the method is called `toGlobalPath`.

I wonder how that combination of driveName:localPath should be called?

Reply:
Ah this method was copied from here for now:

https://github.com/jupyterlab/jupyterlab/blob/897418f0f25ffbbee3aa06a62770a53246823f85/packages/services/src/contents/index.ts#L974-L990

Maybe this should be made reusable. Although we probably want to avoid adding a new method to the interface to avoid breaking changes if that new method is not optional.

Reply:
I would say let's name it as it should (toDriveURI, or whatever) and make it public as optional for V4, making it mandatory for V5?

Reply:
Having a second look at it, we could also just use a different method here since it would only be used by the browser widget. This would likely help keep things simpler to have this patch in a `4.0.x` release.

Also the one in the services package takes a `Contents.IDrive` as a first a parameter instead of a `string`.

But agree we should try to improve that, so normalizing the drive paths becomes accessible to other plugins. Happy to open an issue about this.

Reply:
Don't want to block this PR so OK to address only the BrowserWidget. 

We already have https://github.com/jupyterlab/jupyterlab/issues/15289 that is looking more toward kernels, we may rename it to have a broader discussion, including the drive paths nomenclature.